pyeapi/api/bgp.py

```diff
@@ -78,7 +78,7 @@ def get(self):
         return response
 
     def _parse_bgp_as(self, config):
-        match = re.search(r'^router bgp (\d+)', config)
+        match = re.search(r'^router bgp (\d+.\d+)', config)
         return dict(bgp_as=int(match.group(1)))
```

Comment:
Line 82 converts to integer though so even if the regex allows the dot, converting it to int will cause an the script to crash and throw an exception:

    $ python3
    >>> string = "65001.10"
    >>> print(int(string))
    Traceback (most recent call last):
      File "<stdin>", line 1, in <module>
    ValueError: invalid literal for int() with base 10: '65001.10'

Reply:
Yep, been discussing this on reddit and well and just noticed. Hoped that to be a quick fix, but this seems to be ahead me. So I guess this can be closed and somebody with more experience should try getting this to work, or confirm that this (returning also asdot) will not be implemented so I have to get asdot notation in a different way somewhere in my script.

Reply:
How about if you handle it like this:
```python
match = re.search(r'^router bgp (\d+)\.?(\d+)?', config)
if match.lastindex == 2:
   return dict(bgp_as='.'.join(match.group(1,2)) )
else:
   return dict(bgp_as=int(match.group(1)))
```
if it is dotASN you return it as a string, and otherwise you'd return the int as currently exists

Reply:
Thanks for the replies, I have updated and tested again and it seems to work now. Do I have to re-open the pull request or is this automatically added?